redbeams/src/main/java/com/sequenceiq/redbeams/converter/spi/DBStackToDatabaseStackConverter.java

```diff
@@ -227,10 +227,22 @@ private Optional<String> getGcpEncryptionKeyFromEnv(DetailedEnvironmentResponse
     }
 
     private String getEncryptionKeyResourceGroupNameFromEnv(DetailedEnvironmentResponse environment) {
-        return Optional.ofNullable(environment)
+        /* There would not be a case where both encryptionKeyResourceGroupName and azure Resource group is null.
+        *  If multiple azure resource groups then encryptionKeyResourceGroupName is mandatorily required to enable encryption with CMK.
+        *  This is already checked during environment start as part of azureEncryptionResources.
+        *  At least one of --resource-group-name or --encryption-key-resource-group-name should be specified.
+        * */
```

Comment:
Agreed. The only thing missing is an UT case for this. 😃  Could you please add one to cover the RG name fallback?